src/ott/problems/linear/potentials.py

```diff
@@ -157,6 +161,139 @@ def tree_unflatten(
   ) -> "DualPotentials":
     return cls(*children, **aux_data)
 
+  def plot_ot_map(
```

Comment:
can we imagine putting this in the tools section of the code? feels a bit out of place in this context, or is there a good reason to have it inside the linear problems area, WDYT?

Reply:
Yeah, I'm okay with them over there with a name like `plot_W2_{map,potentials}`. How about you @michalk8?

Reply:
Yes, `ott/tools/potentials.py` sounds good. As for renaming, don't think we should add the W2; the `plot_ot_map/plot_potential` is ok.
Also for consistency, I'd rename `inverse` argument to `forward`.

Reply:
What about `plot_neural_ot_map` and `plot_neural_potential` to clarify it's not for the discrete/sinkhorn maps?

Reply:
These methods are available to the `EntropicMap`, therefore `neural` won't work.

Reply:
Ah, then `plot_contunous_ot_map` and `plot_continuous_potential`? Or do you want to keep the shorter names?

Reply:
maybe continuous is not needed and shorter is ok?

Reply:
Ok, adding that now